central/networkgraph/flow/datastore/internal/store/postgres/store.go

```diff
@@ -98,6 +110,10 @@ var (
 	// to deal with failures if we just sent it all.  Something to think about as we
 	// proceed and move into more e2e and larger performance testing
 	batchSize = 10000
+
+	deleteTimeout = 3 * time.Minute
```

Comment:
Since we currently do not have a better way to do this, please make these configurable env vars